src/main/java/analyzer/analyzerInterfaces/AnalyzerComment.java

```diff
@@ -0,0 +1,22 @@
+package analyzer.analyzerInterfaces;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+public interface AnalyzerComment {
+
+    // Constructed message with all the template variables applied.
+    String message();
+
+    // The message with template variables in there.
+    String template();
```

Comment:
Do you need these two fields? They require you to get the values from the `website-copy` repo and you don't need them for the analyzer output.

Side note: I could see this being useful for some type of integration test though (but maybe not define it here then).

Reply:
Yes for the most part I don't think we need these now that I had more time to think about it after I had created the PR. I will replace these. Like I mentioned the only thing I have to go off of is the JS analyzer. @ErikSchierboom